.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.env
\ No newline at end of file
+.env
+.DS_Store
```

Comment:
FWIW, I think it's worthwhile to add `.DS_Store` to your `~/.gitignore` so that these platform specific files don't have to be added to the project `.gitignore`.

Reply:
hmmm this is already in my ~/.gitignore.  i wonder why it's not working